.env

```diff
@@ -46,12 +46,12 @@ LOCAL_DB_TYPE=mysql
 ##
 # The database version to use.
 #
-# Defaults to 5.7 with the assumption that LOCAL_DB_TYPE is set to `mysql` above.
+# Defaults to 8.0 with the assumption that LOCAL_DB_TYPE is set to `mysql` above.
 #
 # When using `mysql`, see https://hub.docker.com/r/amd64/mysql for valid versions.
 # When using `mariadb`, see https://hub.docker.com/r/amd64/mariadb for valid versions.
```

Comment:
Given https://github.com/WordPress/wordpress-develop/pull/5834/files#r1455897013, we might as well point to the platform-independent reference:

```suggestion
# When using `mysql`, see https://hub.docker.com/_/mysql for valid versions.
# When using `mariadb`, see https://hub.docker.com/_/mariadb for valid versions.
```